cmp/grid/columns/Column.ts

```diff
@@ -455,7 +462,7 @@ export class Column {
     autosizeBufferPx: number;
     autoHeight: boolean;
     editable: boolean | ColumnEditableFn;
-    editor: ColumnEditorFn;
```

Comment:
Thanks for fixing this!

Reply:
don't thank me, thank typescript -- it insisted!